frontend/src/host_orchestrator/orchestrator/createcvdaction.go

```diff
@@ -148,6 +154,9 @@ func (a *CreateCVDAction) launchWithCanonicalConfig(op apiv1.Operation) (*apiv1.
 			args = append(args, "--credential_source=gce")
 		}
 	}
+	if a.buildAPICredentials.CredsUserProjectID != "" {
+		args = append(args, "--project_id="+a.buildAPICredentials.CredsUserProjectID)
+	}
```

Comment:
Should we set `--project_id` only when `a.buildAPICredentials.AccessToken != ""`?